data/Chinwenma.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "Chinwenma Okorie",
+  "bio": " **FrontEnd Developer** | Open Source enthusiast | Blogger | Tech enthusiast",
+  "tags":[ " Open Source, NextJs, ReactJs,  JavaScript, Tailwind, Html5, CSS3"],
```

Please fix the array syntax here.

```suggestion
  "tags":[ " Open Source", "NextJs", "ReactJs",  "JavaScript", "Tailwind", "Html5", "CSS3"],
```

---

ok thanks